cloudfour/cloudfour.com-patterns
# Basic size and scale tokens

## Overview

Introduces design tokens for modular scale and misc. size values. Also introduces a modular scale function for Sass. I found cases where these would be useful while working on #475 and #470.

There are a handful of existing modular scale libraries for Sass, but none of them seem to have been written since Sass added built-in support for exponential math. I found I was able to recreate 99% of their functionality in a single line.

The documentation pages are a work in progress but at least make the existence of the tokens known.

## Screenshots

<img width="828" alt="Screen Shot 2020-03-04 at 9 18 10 AM" src="https://user-images.githubusercontent.com/69633/75905295-79cded00-5df9-11ea-94c2-1560512d8ede.png">

## Testing

1. View deploy preview
1. Observe tweaks to demo button using values from new tokens
1. Observe existence of "Design Tokens/Size and Scale" page
